packages/database/src/getDbMigrator.js

```diff
@@ -0,0 +1,54 @@
+/**
```

Comment:
Most of this can be diff'd locally against `migrate.js`. The only differences are the different callback signature when running it from the app rather than cli, and the change of migration directory (which is specified on the command line when run via yarn anyway)